server/lib/tuist/github/releases.ex

```diff
@@ -26,24 +26,32 @@ defmodule Tuist.GitHub.Releases do
   end
 
   def get_latest_cli_release(opts \\ []) do
-    case opts
-         |> fetch_releases()
-         |> Enum.find(fn release ->
-           not String.contains?(release["name"], "@")
-         end) do
-      nil -> nil
-      release -> map_release(release)
+    if Tuist.Environment.dev?() do
```

Comment:
I was consistently getting errors due to a rate limit in my local server and I don't think we need to run this locally (same for `get_latest_app_release`)